nx/public/sl/components.css

```diff
@@ -152,14 +170,88 @@ button {
     outline-offset: 4px;
   }
 
+  &:not(:disabled) {
+    cursor: pointer;
+  }
+
+  &.negative {
+    background: var(--s2-red-900);
+    border-color: var(--s2-red-900);
+    outline-color: var(--s2-red-900);
+
+    &:not(:disabled):hover {
+      background: var(--s2-red-800);
+      border-color: var(--s2-red-800);
+    }
+  }
+
   &.primary {
     background: var(--s2-gray-800);
     border: 2px solid var(--s2-gray-800);
     color: #fff;
 
+    &:not(:disabled):hover {
+      background: var(--s2-gray-900);
+      border-color: var(--s2-gray-900);
+    }
+
     &.outline {
       background: transparent;
       color: var(--s2-gray-800);
+
+      &:not(:disabled):hover {
+        background: transparent;
+        border-color: var(--s2-gray-600);
+        color: var(--s2-gray-600);
+      }
     }
   }
+
+  &:disabled {
+    opacity: 0.6;
+  }
+
+  &:not(:disabled):hover {
+    background: var(--s2-blue-800);
+    border-color: var(--s2-blue-800);
+  }
+}
+
+.sl-dialog {
+  --dialog-font-size: 16px;
+
+  max-width: 95vw;
+  box-sizing: border-box;
+  border-radius: var(--s2-radius-300);
+  display: flex;
+  border: none;
+  justify-content: end;
+  flex-direction: column;
+  padding: var(--spacing-500);
+  font-family: var(--body-font-family);
```

Comment:
This is mostly all good. We definitely want to ship this dialog. Just with one slot that consumes any children.

My only nit: we should probably go with a fixed width so things don't blow out if someone develops in a stand alone page (something I did early on).

Reply:
WDYT? `max-width: min(95vw, 600px)`